src/iterator.rs

```diff
@@ -31,8 +29,8 @@ impl PointLocation {
     pub fn get_point_culling(&self) -> Box<dyn PointCulling<f64>> {
         match &self {
             PointLocation::AllPoints => Box::new(AllPoints {}),
-            PointLocation::Aabb(aabb) => Box::new(*aabb),
-            PointLocation::Frustum(frustum) => Box::new(frustum.clone()),
+            PointLocation::Aabb(aabb) => Box::new(aabb.clone()),
```

Comment:
nit: remove clone if you choose to derive `Copy`

Reply:
I'm resolving those – if we implement `Copy`, which I wouldn't unless you have a clear preference, clippy will remind me to do that anyway.